tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/TypeTestExprTest.java

```diff
@@ -48,6 +48,22 @@ public void testTypeTestExprSemanticsNegative() {
         BAssertUtil.validateError(negativeResult, 0, "unknown type 'C'", 35, 14);
     }
 
+    @Test
```

Comment:
Shouldn't this tests be added in org.ballerinalang.test.statements.ifelse.TypeGuardTest instead?

Reply:
moved